apache/superset
# fix(emotion): add separate namespace for menu app

### SUMMARY

Adds emotion cache to the menu "app" so that styles do not collide with other other apps. The cache allows specifying a namespace for the emotion styles that is seperate from the default namespace, used by other "apps" on the same page.

 More info on the issue here: https://github.com/emotion-js/emotion/issues/2210

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

before:
<img width="1681" alt="Screen Shot 2021-04-26 at 6 02 15 PM" src="https://user-images.githubusercontent.com/10255196/116169469-f8eb4800-a6c1-11eb-9fa9-f6a8b831b5a8.png">
after:
<img width="1677" alt="Screen Shot 2021-04-26 at 6 03 00 PM" src="https://user-images.githubusercontent.com/10255196/116169503-0a345480-a6c2-11eb-8b8f-dd694f71e788.png">

### TEST PLAN

- run `npm run build`
- see proper menu styling on the sqllab/dashboard/explore pages. 

### ADDITIONAL INFORMATION
